examples/java/spring-boot/manifest.yml

```diff
@@ -0,0 +1,7 @@
+---
+applications:
+  - name: federated-api-model-spring-boot-sandbox
+    buildpacks:
+      - https://github.com/cloudfoundry/java-buildpack.git#v4.47
```

Comment:
Turns out this isn't what's used by Spring Boot - running `./gradlew buildBootImage` shows it uses, among others:

```
    [creator]     Paketo Spring Boot Buildpack 5.3.1                                                                                                                      
    [creator]       https://github.com/paketo-buildpacks/spring-boot            
```

Did we want to give that a go, or get it in at least as-is, then move to that in the future?

Reply:
I'll give it a go with this one